src/components/RulesTable.js

```diff
@@ -0,0 +1,135 @@
+        {value.engine}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.locale_regex}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.strategy}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.keyword_regex}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.max_tries}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.proxy_bal}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.dest_scraper}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.overwrite}
+      </TableRowColumn>
+      <TableRowColumn className="td_box">
+        {value.raw_html}
+      </TableRowColumn>
+    </TableRow>
+) // end of SortableItem
+
+const SortableList = SortableContainer(
+  (state) => {
+
+    const {rules} = state
+
+    return (
+
+      <Table className="ruleTable">
+        <TableHeader displaySelectAll = {false} adjustForCheckbox	= {false} >
+          <TableRow
+            rowNumber	= {1}
+            displayBorder = {true}
+            >
+            <TableHeaderColumn tooltip={"Search Engines: like google, bing, yahoo"}>
+              Engine
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"Language - Country of origin"}>
+              Locale Regex
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"What strategy to use"}>
+              Strategy
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"Who uses this??"}>
+              Keyword Regex
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"0 - 100"}>
+              max_tries
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"What Proxy Balancer to use"}>
+              proxy_bal
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"Scraper that will be used"}>
+              dest_scraper
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"No idea what this is."}>
+              overwrite
+            </TableHeaderColumn>
+            <TableHeaderColumn tooltip={"hardly ever used."}>
+              raw_html
+            </TableHeaderColumn>
+          </TableRow>
+        </TableHeader>
+        <TableBody className="ruleTable"
+          showRowHover = {true}
+
+          >
+            {rules.map((value, index) => (
+              <SortableItem key={`rule-${index}`} index={index} value={value} />
+            ))}
+        </TableBody>
+    </Table>
+
+    );
+  }
+) // end of SortableList
+
+class SortableComponent extends Component {
+
+  onSortEnd = ({oldIndex, newIndex}) => {
+     this.setState({
+       rules: arrayMove(this.props.rules, oldIndex, newIndex),
+     });
+   };
+   render() {
+    const currentEngineFilter = this.props.engine // comes from a reducer (MapStateToProps)
```

Comment:
It's good to consistently use `;` at the end of each line of JS.  JavaScript made them optional, but it was a mistake.  In regular JS, you can add `"use strict";` to the top of the file to enforce things like this.  I think this file is JSX, so that might not work.